vscode/src/package.ts

```diff
@@ -0,0 +1,38 @@
+
+export interface ConsolePackage {
+  // The JSON from console does not follow the naming-convention.
+  /* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Actually this one one does.
I'm fine with disabling the linter, though. Just in case we change it in the future.